src/lib/program.ts

```diff
@@ -171,6 +171,10 @@ iam
   .option('-a, --account <alksAccount>', 'alks account to use')
   .option('-r, --role <alksRole>', 'alks role to use')
   .option('-F, --favorites', 'filters favorite accounts')
+  .option(
+    '--tags <tags...>',
+    `A list of resource tags. Keys are uniquely identifiable and values can contain a comma separated list . Can either be a JSON representation '[{"Key":"key1","Value":"value1"},{"Key":"key2","Value":"value2,value3"}]' or in shorthand Key=key1,Value=value1 Key=key2,Value=value2,value3`
```

Comment:
Confirmed that this would cause an error if someone includes a comma in the tag value.  AWS cli or core api must do some parsing of that, which we don't do.

Reply:
I deleted the example. Other than that, it's not explicitly stated anywhere else